src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Display from './Display';
+
+function App() {
+  return (
+    <fragment>
+      <h1>
+        Milestone #2
+      </h1>
+      <Display value="0" />
+    </fragment>
```

Comment:
- [ ] Please change this to the exact React.Fragment written as such, or consider using the [shorter syntax](https://reactjs.org/docs/fragments.html#short-syntax).